.github/workflows/release.yml

```
@@ -27,7 +27,7 @@ jobs:
       - run: make python-sdk
       - run: python setup.py build sdist
         working-directory: sdk/python
-      - uses: pypa/gh-action-pypi-publish@master
+      - uses: pypa/gh-action-pypi-publish@v1.8.8
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
           packages_dir: sdk/python/dist/
```

looks like this was updated to be hyphenated
https://github.com/pypa/gh-action-pypi-publish#customizing-target-package-dists-directory

```suggestion
          packages-dir: sdk/python/dist/
```